### PR TITLE
Grouping frontend v2

### PR DESCRIPTION
**Pull request by cptbtptp01**

### CODE REVIEW

File: src/assets/notification.wav
Comment (z-q-ying) [inline src/assets/notification.wav:1]: we may add this to .gitignore next time?

**Review by z-q-ying — APPROVED** [review]
Other than the small comment all looks good to me! Thanks a lot for the great efforts!!

File: src/assets/notification.wav
Comment (koolgax99) [reply src/assets/notification.wav:1]: But I think he is using that for the sound

Comment (koolgax99) [reply src/assets/notification.wav:1]: But I am still wondering about how this appeared

**Pull request merged**